src/exact_geodesic.cpp

```diff
@@ -42,6 +43,11 @@ npe_default_arg(fs, npe_matches(f), pybind11::array())
 npe_default_arg(ft, npe_matches(f), pybind11::array())
 npe_begin_code()
 
+  assert_valid_3d_tri_mesh(v, f);
+  //assert_cols_equals(vs, 1, "vs");
+  //assert_cols_equals(fs, 1, "fs");
+  //assert_cols_equals(vt, 1, "vt");
+  //assert_cols_equals(ft, 1, "ft");
```

Comment:
Why are these commented out? 

Also, I'm not sure if these will always have shape (N, 1). They might have shape (1, N). It might make sense to have something like `assert_is_vector(vs, "vs")` which checks both conditions.

Reply:
True, these ones I'm not sure (I'm not even sure if they can be omitted), so I added them but comment them out.